hammurabi/rules/text.py

```diff
@@ -68,6 +68,7 @@ def __init__(
         target: Optional[str] = None,
         position: int = 1,
         respect_indentation: bool = True,
+        trailing_newline: bool = False,
```

Comment:
minor: I would name this as `add_trailing_newline`

Reply:
How about `ensure_trailing_newline`?

Reply:
even better